### PR TITLE
Update Set-Label.md

### PR DESCRIPTION
**Pull request by SethF-MSFT**

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-Label.md [hunk 1]
```diff
@@ -260,6 +260,8 @@ The ApplyContentMarkingFooterMargin parameter specifies the size (in points) of
 
 This parameter is meaningful only when the ApplyContentMarkingFooterEnabled parameter value is either $true or $false.
 
+Note: In Word, the specified value is used as a bottom margin and left/right margin for left/right aligned content marks. A minimum value of 15 points is required and Word will also add a constant offset of 5 points to the left margin (for left aligned content marks) or right margin (for right aligned content marks).
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-Label.md:263]:
```suggestion
**Note**: In Word, the specified value is used as a bottom margin and left or right margin for left-aligned or right-aligned content marks. A minimum value of 15 points is required and Word will also add a constant offset of 5 points to the left margin (for left-aligned content marks) or right margin (for right-aligned content marks).
```

File: exchange/exchange-ps/exchange/Set-Label.md [hunk 2]
```diff
@@ -391,6 +393,8 @@ The ApplyContentMarkingHeaderMargin parameter specifies the size (in points) of
 
 This parameter is meaningful only when the ApplyContentMarkingHeaderEnabled parameter value is either $true or $false.
 
+Note: In Word, the specified value is used as a top margin and left/right margin for left/right aligned content marks. A minimum value of 15 points is required and Word will also add a constant offset of 5 points to the left margin (for left aligned content marks) or right margin (for right aligned content marks).
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-Label.md:396]:
```suggestion
**Note**: In Word, the specified value is used as a top margin and left or right margin for left-aligned or right-aligned content marks. A minimum value of 15 points is required and Word will also add a constant offset of 5 points to the left margin (for left-aligned content marks) or right margin (for right-aligned content marks).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SethF-MSFT, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@SethF-MSFT Gentle reminder

**Comment by cabailey** [comment]
Don't forget to also update New-Label to keep the information in sync!

**Pull request merged**